src/main/java/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisher.java

```diff
@@ -280,6 +307,15 @@ private boolean isConfigValid(PrintStream logger) throws IOException, Interrupte
             }
         }
 
+        // Check if inAppUpdatePriority is a valid number if not null
+        if (inAppUpdatePriority != null) {
+            try {
+                Integer.parseInt(inAppUpdatePriority);
```

Comment:
This isn't parsing the expanded form, so if it was a variable this will fail, even if the expanded form is valid

Reply:
Fixed in my last commit [a7ea210](https://github.com/jenkinsci/google-play-android-publisher-plugin/pull/37/commits/a7ea210b1e37c9fede38ea05d6ea0d8eeccf1345)